mixer/adapter/stackdriver/internal/cloud.google.com/go/contextgraph/apiv1alpha1/mock_test.go

```diff
@@ -16,10 +16,6 @@
 
 package contextgraph
 
-import (
-	contextgraphpb "istio.io/istio/mixer/adapter/stackdriver/internal/google.golang.org/genproto/googleapis/cloud/contextgraph/v1alpha1"
```

Comment:
please revert at some point. you probably have a different version of goimports than CI uses? The same happened to me